### DISCUSSION TITLE
Are cursor and voronoi containers supported with custom multi-chart charts?

### DISCUSSION THREAD

**Discussion by johnnywang** [FormidableLabs/victory#2116] [Q&A]
# Bugs and Questions

### Checklist

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [ ] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

More a question because it's not clear from the documentation: when creating a custom chart with multiple, layered chart types (i.e. line + scatter + area), is it possible to still use the cursor and/or voronoi containers? We're currently using a custom cursor + voronoi with an area chart, and the container documentation seems to suggest we'd lose this ability if we combined multiple charts.

**Comment by becca-bailey** [comment]
Hi @johnnywang, I'm going to move this issue to a discussion thread so we can continue to discuss it there.
